python/pyspark/ml/feature.py

```diff
@@ -2629,8 +2629,28 @@ class ChiSqSelector(JavaEstimator, HasFeaturesCol, HasOutputCol, HasLabelCol, Ja
     """
     .. note:: Experimental
 
-    Chi-Squared feature selection, which selects categorical features to use for predicting a
-    categorical label.
+    Creates a ChiSquared feature selector.
```

Comment:
Please add back in the fact that this selects categorical features for predicting a categorical label.  It's useful to state expected input data types.